tests/src/core/getExports.js

```diff
@@ -11,6 +11,12 @@ import * as fs from 'fs';
 import { getFilename } from '../utils';
 import { test as testUnambiguous } from 'eslint-module-utils/unambiguous';
 
+const doesBabelHaveVisitorKeys = () => {
+  const babelPath = require.resolve('babel-eslint');
```

Comment:
is there a reason not to do this at module level, and also use it on line 30?

Reply:
this is just cherry-picked from #3062 so you should put this as a review on that 🙂